presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoParameterMetaData.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.jdbc;
+
+import com.google.common.collect.ImmutableList;
+
+import java.sql.ParameterMetaData;
+import java.sql.SQLException;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class PrestoParameterMetaData
+        extends AbstractMetaData implements ParameterMetaData
```

Comment:
Formatting nit: `extends` and `implements` should be on separate lines. See `PrestoPreparedStatement`.
```java
public class PrestoParameterMetaData
        extends AbstractMetaData
        implements ParameterMetaData